palindrom-redirect.html

```diff
@@ -46,7 +50,7 @@
                 history.pushState(null, null, url);
                 this.dispatchEvent(
                     new CustomEvent(
-                      `${this.getAttribute('is')}-pushstate`,
+                      `${this.localName}-pushstate`,
```

Comment:
This can be changed to static `palindrom-redirect-pushstate` now backward compat is killed.